ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -391,65 +391,68 @@ router bgp {{ router_bgp.as }}
+{%             if router_bgp.vrfs[vrf].neighbors[neighbor].remote_as is arista.avd.defined %}
       neighbor {{ neighbor }} remote-as {{ router_bgp.vrfs[vrf].neighbors[neighbor].remote_as }}
-{%                 elif router_bgp.vrfs[vrf].neighbors[neighbor].peer_group is arista.avd.defined %}
+{%             endif %}
+{%             if router_bgp.vrfs[vrf].neighbors[neighbor].peer_group is arista.avd.defined %}
       neighbor {{ neighbor }} peer group {{ router_bgp.vrfs[vrf].neighbors[neighbor].peer_group }}
-{%                 endif %}
-{%                 if router_bgp.vrfs[vrf].neighbors[neighbor].local_as is arista.avd.defined and router_bgp.vrfs[vrf].neighbors[neighbor].local_as is number %}
+{%             endif %}
+{%             if router_bgp.vrfs[vrf].neighbors[neighbor].local_as is arista.avd.defined %}
       neighbor {{ neighbor }} local-as {{ router_bgp.vrfs[vrf].neighbors[neighbor].local_as }} no-prepend replace-as
-{%                 endif %}
-{%                 if router_bgp.vrfs[vrf].neighbors[neighbor].description is arista.avd.defined %}
+{%             endif %}
+{%             if router_bgp.vrfs[vrf].neighbors[neighbor].description is arista.avd.defined %}
       neighbor {{ neighbor }} description {{ router_bgp.vrfs[vrf].neighbors[neighbor].description }}
+{%             endif %}
+{%             if router_bgp.vrfs[vrf].neighbors[neighbor].ebgp_multihop is arista.avd.defined %}
+{%                 set neighbor_ebgp_multihop_cli = "neighbor " ~ neighbor ~ " ebgp-multihop" %}
+{%                 if router_bgp.vrfs[vrf].neighbors[neighbor].ebgp_multihop is number %}
+{%                     set neighbor_ebgp_multihop_cli = neighbor_ebgp_multihop_cli ~ " " ~ router_bgp.vrfs[vrf].neighbors[neighbor].ebgp_multihop %}
 {%                 endif %}
-{%                 if router_bgp.vrfs[vrf].neighbors[neighbor].ebgp_multihop is arista.avd.defined %}
-      neighbor {{ neighbor }} ebgp-multihop {{ router_bgp.vrfs[vrf].neighbors[neighbor].ebgp_multihop if router_bgp.vrfs[vrf].neighbors[neighbor].ebgp_multihop is number else none }}
-{%                 endif %}
-{%                 if router_bgp.vrfs[vrf].neighbors[neighbor].next_hop_self is arista.avd.defined(true) %}
+      {{ neighbor_ebgp_multihop_cli }}
+{%             endif %}
+{%             if router_bgp.vrfs[vrf].neighbors[neighbor].next_hop_self is arista.avd.defined(true) %}
       neighbor {{ neighbor }} next-hop-self
-{%                 endif %}
-{%                 if router_bgp.vrfs[vrf].neighbors[neighbor].timers is arista.avd.defined %}
+{%             endif %}
+{%             if router_bgp.vrfs[vrf].neighbors[neighbor].timers is arista.avd.defined %}
       neighbor {{ neighbor }} timers {{ router_bgp.vrfs[vrf].neighbors[neighbor].timers }}
+{%             endif %}
+{%             if router_bgp.vrfs[vrf].neighbors[neighbor].send_community is defined or router_bgp.vrfs[vrf].neighbors[neighbor].send_community is arista.avd.defined("none") %}
```

Comment:
Previous code checked for "None" specifically. So capital N. Should we add a "| lower" filter ?

Reply:
The logic matches the old datamodel, but it is still very clunky because we try to allow the `None`/`null` value

Reply:
updated logic to match the current data model and behavior see commit: 13048bf
As per our discussion, I will open an issue to refactor send-community for both global and vrf definition within router-bgp.

Reply:
tracked under issue #706